spec/controllers/hub/outbound_calls_controller_spec.rb

```diff
@@ -58,10 +58,11 @@
     context "with a logged in user" do
       before { sign_in user }
 
-      it "updates the outbound call with the note body" do
+      it "updates the outbound call with the note body and redirects back to messages" do
         put :update, params: params
 
         expect(outbound_call.reload.note).to eq "I talked to them!"
+        expect(response).to redirect_to(hub_client_messages_path(client_id: client.id, anchor: "last-item"))
```

Comment:
we didn't previously have an expectation about where the user would be redirected after submitting an outbound phone call form, but we were redirecting to the notes tab. This adds the expectation and corrects it to redirect to messages instead, where we now display outbound calls.